### ISSUE TITLE
[Question]: anyone reporting bluebird issues after the version bump to 0.7.0?

### ISSUE THREAD

**Issue by toranb** [testem/testem#904]
First up -thanks for all your time maintaining this project! I am in your debt from the start as an ember-cli user :) so thank you!!!

I realized recently that on change I get an error referencing bluebird in the stack as of testem 1.9.0

https://github.com/testem/testem/commit/bd03b4af8db9f91585566364fd41623275831025

I found the commit above did a version bump and I'm curious if anyone else is having this problem? I'm using ember-cli 2.4.2 and node v4.4.5 and testem 1.9.0

```
Error: Run canceled.
    at EventEmitter.<anonymous> (/Users/tbillups/work/wat/zooapp/node_modules/ember-cli/node_modules/testem/lib/app.js:355:32)
    at tryCatcher (/Users/tbillups/work/wat/zooapp/node_modules/ember-cli/node_modules/testem/node_modules/bluebird/js/release/util.js:16:23)
    at MappingPromiseArray._promiseFulfilled (/Users/tbillups/work/wat/zooapp/node_modules/ember-cli/node_modules/testem/node_modules/bluebird/js/release/map.js:57:38)
    at MappingPromiseArray.PromiseArray._iterate (/Users/tbillups/work/wat/zooapp/node_modules/ember-cli/node_modules/testem/node_modules/bluebird/js/release/promise_array.js:113:31)
    at MappingPromiseArray.init (/Users/tbillups/work/wat/zooapp/node_modules/ember-cli/node_modules/testem/node_modules/bluebird/js/release/promise_array.js:77:10)
    at new MappingPromiseArray (/Users/tbillups/work/wat/zooapp/node_modules/ember-cli/node_modules/testem/node_modules/bluebird/js/release/map.js:25:10)
    at map (/Users/tbillups/work/wat/zooapp/node_modules/ember-cli/node_modules/testem/node_modules/bluebird/js/release/map.js:152:12)
    at Function.Promise.map (/Users/tbillups/work/wat/zooapp/node_modules/ember-cli/node_modules/testem/node_modules/bluebird/js/release/map.js:160:12)
    at EventEmitter.App.singleRun (/Users/tbillups/work/wat/zooapp/node_modules/ember-cli/node_modules/testem/lib/app.js:353:21)
    at /Users/tbillups/work/wat/zooapp/node_modules/ember-cli/node_modules/testem/lib/app.js:144:32
    at tryCatcher (/Users/tbillups/work/wat/zooapp/node_modules/ember-cli/node_modules/testem/node_modules/bluebird/js/release/util.js:16:23)
    at /Users/tbillups/work/wat/zooapp/node_modules/ember-cli/node_modules/testem/node_modules/bluebird/js/release/using.js:185:26
    at tryCatcher (/Users/tbillups/work/wat/zooapp/node_modules/ember-cli/node_modules/testem/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/Users/tbillups/work/wat/zooapp/node_modules/ember-cli/node_modules/testem/node_modules/bluebird/js/release/promise.js:504:31)
    at Promise._settlePromise (/Users/tbillups/work/wat/zooapp/node_modules/ember-cli/node_modules/testem/node_modules/bluebird/js/release/promise.js:561:18)
    at Promise._settlePromise0 (/Users/tbillups/work/wat/zooapp/node_modules/ember-cli/node_modules/testem/node_modules/bluebird/js/release/promise.js:606:10)
    at Promise._settlePromises (/Users/tbillups/work/wat/zooapp/node_modules/ember-cli/node_modules/testem/node_modules/bluebird/js/release/promise.js:685:18)
    at Promise._fulfill (/Users/tbillups/work/wat/zooapp/node_modules/ember-cli/node_modules/testem/node_modules/bluebird/js/release/promise.js:630:18)
```

**Comment by johanneswuerbach** [comment]
1.9.0 introduced a serious and unfixed issue, please upgrade to 1.9.1 until the issue has been fixed. 1.9.1 is just a republished 1.8.1, but should prevent anyone ending up with a broken install until the issue has been resolved.

**Comment by toranb** [comment]
@johanneswuerbach ah I'll give that a try tomorrow :) thanks!

**Comment by john-kurkowski** [comment]
Seeing this in testem v1.10.1. Also, Node.js v4.2.6, ember-cli v2.5.0.

My tests launch and continuously re-run. I'll tab away to edit code for awhile. I tab back to the terminal to find Testem exited. No obvious pattern yet.

**Comment by sammilechman** [comment]
Seeing this in testem v1.6.0, same pattern as @john-kurkowski

**Comment by johanneswuerbach** [comment]
This should be resolved in v1.10.2
